usaspending_api/references/management/commands/load_gtas.py

```diff
@@ -10,16 +10,33 @@
 logger = logging.getLogger("console")
```

Comment:
Can you change the log formatter to "script" for more details in the log statements? (All ETL scripts should use "script" instead of "console" to provide datetimestamps (with time zone) and log level.

```suggestion
logger = logging.getLogger("script")
```

Reply:
Yes, can do